sklearn/utils/tests/test_estimator_checks.py

```diff
@@ -72,6 +91,15 @@ def test_check_estimator():
     # check that fit does input validation
     msg = "TypeError not raised by fit"
     assert_raises_regex(AssertionError, msg, check_estimator, BaseBadClassifier)
+    # check that sample_weights in fit accepts pandas.Series type
+    try:
+        from pandas import Series
```

Comment:
Raising a skip if there is no pandas requires splitting `test_check_estimator` into more functions. Which may be a good idea, but isn't really pertinent to this issue.

There should be a way to disable the linter catching a line... Look it up?